README.md

```diff
@@ -1,72 +1,41 @@
 # Microsoft Azure Active Directory Authentication Library (ADAL) for Python
+=====================================
 
-The ADAL for python library makes it easy for python applications to authenticate to AAD in order to access AAD protected web resources.
+[![Build Status](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python.svg?branch=dev)](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python)
+[![Documentation Status](https://readthedocs.org/projects/adal-python/badge/?version=latest)](https://adal-python.readthedocs.io/en/latest/?badge=latest)
 
-## Usage
+|[Getting Started](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki)| [Docs](https://aka.ms/aaddev)| [Samples](https://github.com/azure-samples?query=active-directory)| [Support](README.md#community-help-and-support)
+| --- | --- | --- | --- |
 
-### Install
 
-To support 'service principal' with certificate, ADAL depends on the 'cryptography' package. For smooth installation, some suggestions:
+The ADAL for Python library enables python applications to authenticate with Azure AD and get tokens to access Azure AD protected web resources.
 
-* For Windows and macOS
+You can learn in detail about ADAL Python functionality and usage documented in the [Wiki](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki).
 
-  Upgrade to the latest pip (8.1.2 as of June 2016) and just do `pip install adal`.
+## Versions
+Current version - 0.6.0
 
-* For Linux
+Minimum recommended version - 0.6.0
```

Comment:
Again, we probably won't need to hardcode yet another version number here.

On a side note, the idea of "minimum recommended version" is intriguing. What do we want to convey here?

* If the intention is simply want to recommend customers to "always use whatever latest version is available" (which makes perfect sense), we can just write something like that, without hardcoding a specific version number.
* By definition of [Semantic Versioning](https://semver.org/), customers who using an old version X.Y.Z can always safely upgrade to newer version "X.Y-latest.Z-latest", without worrying about breaking change.
* Or customers can technically stay with a version "X.Y.Z", [as long as it still works](https://en.wiktionary.org/wiki/if_it_ain%27t_broke,_don%27t_fix_it).

Reply:
Yes I think specifying that we follow semantic versioning and pointing to the Release page is sufficient.

Reply:
Nitpicking: since we explicitly mention Semantic Versioning in our README, how about we add a link to its official website too?